config/routes.js

```diff
@@ -7,6 +7,7 @@ const locationValidator = require('../app/middleware/locationValidator');
 const logZeroResults = require('../app/middleware/logZeroResults');
 const prerender = require('../app/middleware/prerender');
 const renderer = require('../app/middleware/renderer');
+const debugRenderer = require('../app/middleware/debugRenderer');
```

Comment:
nit: I'm not sure if `Renderer` is required, it could quite easily just be `debug`. And then I think about whether `debug` is a good name. Perhaps `info` would be better?